NEWS.md

```diff
@@ -9,14 +9,40 @@ cvc5 1.2.0
    - Documentation: https://cvc5.github.io/docs-ci/docs-main/api/c/c.html
    - Examples: https://github.com/cvc5/cvc5/tree/main/examples/api/c
 
-- Exposed creation and maintenance of **Skolem functions** to the API.
+- Exposed creation and maintenance of **Skolem functions** to the API. Skolem
+  functions are symbols that are internally introduced by cvc5 during solving.
+  They are formalized via an identifier (see
+  https://cvc5.github.io/docs-ci/docs-main/skolem-ids.html for details) as well
+  as a (possibly empty) set of indices.
+  + The **API** methods `Term::isSkolem()`, `Term::getSkolemId()`, and
+    `Term::getSkolemIndices()`may now be used to identify terms corresponding
+    to skolems.
+
+- Add support for the kind `BITVECTOR_NEGO` corresponding to bitvector
```

Comment:
This was new in 1.1.2, maybe move it there (and also update the release notes for 1.1.2 on GH).

Reply:
Sounds good.